README.md

```diff
@@ -1,215 +1,127 @@
+sudo updatedb
+```
 
-The result of benchmark shows 2x~4x speedup in our typical queries.
+If your GCC version less than 5.3.1, you can run:
 
-## 5.   Backup and Recovery
+```
+sudo yum install devtoolset-4-toolchain -y
+```
 
-Data backup function is provided to enhance data security. The minimum granularity of backup and recovery is partition.  Users can develop plugins to backup data to any specified remote storage. The backup data can always be recovered to Doris at all time, to achieve the data rollback purpose.
+and then, set the path of gcc (e.g /opt/rh/devtoolset-4/root/usr/bin) to the environment variabl PATH.
 
-Currently we only support full data backup data rather than incremental backups for the following reasons:
 
-1. Remote storage system is beyond the control of the Doris system. We cannot guarantee whether the data has been changed between two backup operations. And data verification operations always come at a high price.
+#### Compile and install
 
-2. We support data backup on partition granularity. And majority of applications are time series applications. By dividing data using time column, it has been able to meet the needs of the vast majority of incremental backup in chronological order.
+Run following script, it will comiple thirdparty libraries and build whole Doris.
 
-In addition to improving data security, the backup function also provides a way to export the data. Data can be exported to other downstream systems for further processing.
+```
+sh build.sh
+```
 
-# Install
-Doris only supports Linux System. Oracle JDK 8.0+ and GCC 4.8.2+ are required. See the document of [INSTALL](https://github.com/apache/incubator-doris/wiki/Doris-Install) and [Deploy & Update](https://github.com/apache/incubator-doris/wiki/Doris-Deploy-%26-Upgrade)
+After successful build, it will install binary files to the path of output/.
 
-# User Guide
-See the [Doris Wiki](https://github.com/apache/incubator-doris/wiki) for more information.
+## 5. Reporting Issues
 
-# Contact us
-<dev@doris.apache.org>
+If you find any bugs, please file a [GitHub issue](https://github.com/apache/incubator-doris/issues).
 
-需要加入Doris微信技术讨论群的,请加微信号：morningman-cmy, 然后备注一下：加入Doris技术讨论群
-（Doris开源一群已满）
+## 6. Links
 
-# Blogs
-1. [浅谈从Google Mesa到Apache Doris](http://neoremind.com/2017/09/%E6%B5%85%E8%B0%88%E4%BB%8Egoogle-mesa%E5%88%B0%E7%99%BE%E5%BA%A6palo/comment-page-1/)
-2. [Apache Kylin VS Apache Doris](https://blog.bcmeng.com/post/apache-kylin-vs-baidu-palo.html)
+* Doris official site - <http://doris.incubator.apache.org>
+* User Manual (GitHub Wiki) - <https://github.com/apache/incubator-doris/wiki>
+* Developer Mailing list - Subscribe to <dev@doris.incubator.apache.org> to discuss with us.
+* Gitter channel - <https://gitter.im/apache-doris/Lobby> - Online chat room with Doris developers.
+* Overview - <https://github.com/apache/incubator-doris/wiki/Doris-Overview>
```

Comment:
Overview - <https://github.com/apache/incubator-doris/wiki/Doris-Overview>
add it to the frontend of README.md?